### DISCUSSION TITLE
Delay on upgrade when new release available

### DISCUSSION THREAD

**Discussion by matbe81** [docusealco/docuseal#278] [General]
When
`running sudo docker pull docuseal/docuseal`
it say 
```
Status: Image is up to date for docuseal/docuseal:latest
docker.io/docuseal/docuseal:latest
```
while new version is released.
It look there is always 1-2 days delay for avaialbility.

How to avoid that and be able to upgrade immediately?

Thank you

**Comment by omohokcoj** [comment]
Hi @matbe81 - you can follow the build status here: https://github.com/docusealco/docuseal/actions/runs/9256979906/job/25464158869
Docker builds are triggered by a github releases - that's why there is a delay

**Reply by matbe81** [reply]
Thank you. 
For the moment, I subscribed to github releases notifications.
How to be notified when docker releases is available please?
I'm also interested in an auto/manual upgrade from the GUI...
